javascript/express/security/audit/xss/mustache/explicit-unescape.yaml

```diff
@@ -35,4 +36,4 @@ rules:
     - '*.html'
   pattern-either:
   - pattern-regex: '{{{((?!include).)*?}}}'
-  - pattern-regex: '{{[^}&]*&[^}&]*}}'
+  - pattern-regex: '{{&.*}}'
```

Comment:
I've tested the `{{ & var}}` case on the [Mustache demo](https://mustache.github.io/#demo) and Mustache seems to be OK with it by ignoring all the whitespace characters, thus it won't be detected (https://semgrep.dev/s/4gl9).
Maybe it would be better to use `"{{[\\s]*&.*}}"` or `pattern-inside: "{{&...}}"` using `languages: generic`?

Reply:
Sorry for the delay with this! Thanks for your helpful playground link. I like your suggestion to add the `\s` to capture any optional whitespace.